daml-foundations/daml-ghc/daml-prim-src/GHC/Show.daml

```diff
@@ -21,97 +21,55 @@ import GHC.CString (fromString)
 import GHC.Tuple ()  -- We're using tuples but they are builtin syntax.
 import GHC.Types
 
--- | The `shows` functions return a function that prepends the
--- output 'String' to an existing 'String'.  This allows constant-time
+-- | `showS` is a function that prepends the
+-- output `Text` to an existing `Text`.  This allows constant-time
```

Comment:
This is pretty tricky to document since we can only document the _intent_ of a function of type `ShowS`. Maybe something like:
A function of type `ShowS` _should_ represent some text and applying it so some argument _should_ prepend the argument to the represented text.